hardhat.config.ts

```diff
@@ -43,13 +43,20 @@ export default {
     sources: "src/contracts",
   },
   solidity: {
-    version: "0.7.5",
-    settings: {
-      optimizer: {
-        enabled: true,
-        runs: 1000000,
+    compilers: [
+      {
+        version: "0.7.5",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000000,
+          },
+        },
       },
-    },
+      {
+        version: "0.6.12",
```

Comment:
Any reason we don't want to turn on the optimizer here?

Reply:
Actually, this shouldn't be needed anymore, since you are using Solidity v0.7.5 compatible `StorageAccessible` right?

Reply:
Actually, this is no longer needed since I have jumped all the way to v0.7. Nice unexpected catch!